src/scripts.js

```diff
@@ -1 +1,33 @@
-console.log('Hello world');
+let recipeRepository; 
+
+const recipeCarousel = document.querySelector('.recipe-carousel')
+const searchBox = document.querySelector('.search-box');
+
+window.addEventListener('load', compileRecipeRepository)
+window.addEventListener('load', populateFeaturedRecipesCarousel);
+document.addEventListener('keydown', searchAllRecipes);
+
+function compileRecipeRepository() {
+  recipeRepository = new RecipeRepository(recipeData, ingredientsData)
+}
+
+function populateFeaturedRecipesCarousel() {
+  for (i = 0; i < 5; i++) {
+    let randomRecipe = recipeRepository.recipes[Math.floor(Math.random() * recipeRepository.recipes.length)]
+    recipeCarousel.innerHTML += `
+      <article class="recipe-card">
+            <img class="recipe-card-img" src="${randomRecipe.image}">
+            <p class="recipe-card-name">${randomRecipe.name}</p>
+            <p class="recipe-card-cost">${randomRecipe.getIngredientsCost()}</p>
+            <button class="recipe-card-button">Save Recipe</button>
+      </article>
+    `
+  }
+}
+
```

Comment:
I'm not sure if they want us to use arrow functions over the old functions. We should clarify that.